community/py-pyldap/APKBUILD

```diff
@@ -1,17 +1,15 @@
 # Contributor: Jakub Jirutka <jakub@jirutka.cz>
 # Maintainer: Jakub Jirutka <jakub@jirutka.cz>
 pkgname=py-pyldap
-_pkgname=${pkgname#py-}
-pkgver=2.4.45
-_vermajor=${pkgver%%.*}
-_verminor=${pkgver#*.}; _verminor=${_verminor%%.*}
+_pkgname="python-ldap"
+pkgver=3.1.0
 pkgrel=0
 pkgdesc="Python modules for implementing LDAP clients"
-url="https://github.com/pyldap/pyldap"
+url="https://github.com/python-ldap/python-ldap"
 arch="all"
 license="PSF"
-depends="libldap>=$_vermajor.$_verminor"
-makedepends="python2-dev python3-dev openldap-dev>=$_vermajor.$_verminor"
+depends="libldap"
```

Comment:
This version requires new dependencies, see [setup.py](https://github.com/python-ldap/python-ldap/blob/ec41070d3fa1d00cb272f86e0c8ca55db51bdf62/setup.py#L166-L169):

```
  install_requires=[
    'pyasn1 >= 0.3.7',
    'pyasn1_modules >= 0.1.5',
],
```

Reply:
Fixed, thanks!
See also #5641